1-exercises/A-undefined/exercise.js

```diff
@@ -21,7 +21,7 @@ function sayHello() {
 
 let hello = sayHello();
 console.log(hello);
-
+//function sayHello doesn't have any paramets and there is not argument when it calls !
 
 // Example 3
 function sayHelloToUser(user) {
```

Comment:
Can't see your solution for this one! by using template literals and calling the function, you're able to print out "user's name"

Reply:
Also missing example 4, which is a good to learn how to use retrieve an element from array